cmd/catalog.go

```diff
@@ -184,13 +186,81 @@ services from service catalog.
 	},
 }
 
+var catalogDescribeCmd = &cobra.Command{
+	Use:   "describe",
+	Short: "describe given service.",
```

Comment:
This should probably say something like "Describe catalog item". This message is for generic `odo catalog describe` command.